src/main/kotlin/Locations.kt

```diff
@@ -33,4 +33,14 @@ class Locations {
 
     @Location("/teams/{uuid}")
     data class Team(val uuid: String = "")
+
+    @Location("/create_message")
```

Comment:
This has to be called message

Reply:
just to confirm, it doesn't have to also have a `get` here right? even though it's technically REST convention? since we don't actually ever want to return all messages.